README.md

```diff
@@ -47,6 +47,7 @@ Usage:
 Available Commands:
   completion  generate the autocompletion script for the specified shell
   discover    Discover referrers for a subject
+  resolve     Resolve digest if subject is referenced by a tag
```

Comment:
`if` might seem conditional. I would recommend `resolve digest of a subject that is referenced by a tag`